build/Dockerfile

```
@@ -1,22 +1,22 @@
 # Build stage
 FROM --platform=${BUILDPLATFORM:-linux/amd64} golang:1.15.4-alpine AS builder
-COPY ./.git promscale/.git
-COPY ./pkg promscale/pkg
-COPY ./cmd promscale/cmd
-COPY ./go.mod promscale/go.mod
-COPY ./go.sum promscale/go.sum
+RUN apk update && apk add --no-cache git
+WORKDIR /promscale
+COPY ./go.mod .
+COPY ./go.sum .
```

Those two should be in the same layer as they are changed always at the same time.
```suggestion
COPY go.mod go.sum .
```